lib/perl/Genome/Library.pm

```diff
@@ -154,6 +154,45 @@ sub is_rna {
     return 0;
 }
 
+sub is_10x_atac {
+    my $self = shift;
+
+    return 1 if Genome::Utility::List::in(
+        $self->protocol, (
+            "10x_SC_ATAC_SEQ"
+        )
+    );
+
+    return 0;
+}
+
+sub is_10x_gex {
+    my $self = shift;
+
+    return 1 if Genome::Utility::List::in(
+        $self->protocol, (
+            "10x_SC-5'GEX",
+            "10x_SC-3'GEX V3",
+            "10x_SC-5'GEX"
```

Comment:
This list includes the same item, `10x_SC-5'GEX`, twice.

Reply:
good catch! One of them was supposed to be `10x_SC-3'GEX`. Just pushed the correction.